panacea-3/upgrade-fullnode.md

```diff
@@ -30,11 +30,17 @@ make install
 panacead version --long
 ```
 
-5. Rename the config directory and reset state.
+5. Create a new `~/.panacea` directory and copy previous config files.
 ```bash
-mv ~/.panacead ~/.panacea
+MONIKER=$(grep "^moniker = " ~/panacea-2-backup/.panacead/config/config.toml | awk '{print $3}' | sed 's|"||g')
```

Comment:
`3. Back up your `~/.panacead` directory`
How about providing the backup creation folder name in bash below here?

```bash
mkdir -p ~/panacea-2-backup
cp -r ~/.panacead ~/panacea-2-backup
cp -r ~/.panaceacli ~/panacea-2-backup
```

Reply:
Yeah. I will improve the explanation.

Reply:
Done bd76cea. I made it much simpler and clearer.